shared/trainer/defaults.yaml

```diff
@@ -136,6 +138,7 @@ gen:
     use_minent_var: True # add variance of entropy map in the measure of entropy for a certain picture
     use_advent: True # minimize the entropy of the image by adversarial training
     use_ground_intersection: True
+    do_feat_fusion: false
```

Comment:
why would you have use_dada and not do_feat_fusion?

Reply:
hmmm i've put `use_dada `in gen.d to tell that we're using DADA as presented in the paper (so feat fusion + dada fusion on seg head), and I've put `do_feature_fusion `for the masker instead of use_dada to tell that we're just doing feature fusion (and not dada fusion also). You think I should maybe have `do_feature_fusion `and `do_dada_fusion `in both gen.s and gen.m instead? Wasn't sure how to organize that.

Reply:
oh ok I see dada is for the seg head and do_feat_fusion is for the mask head